scripts/generate-rlmplatform.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
With CocoaPods, `RLMPlatform.h` is always empty. That's handled by `build.sh cocoapods-setup`.